sys-cluster/teleport/files/teleport.confd

```diff
@@ -0,0 +1,30 @@
+# /etc/conf.d/teleport: config file for /etc/init.d/teleport
+
+# Where is your teleport.yaml file stored?
+
+TELEPORT_CONFDIR="/etc/teleport"
+
+
+# Any random options you want to pass to teleport.
+
+TELEPORT_OPTS=""
+
+
+# Pid file to use (needs to be absolute path).
+
+#TELEPORT_PIDFILE="/var/run/teleport.pid"
+
+
+# Path to the teleport binary (needs to be absolute path).
+
+#TELEPORT_BINARY="/usr/bin/teleport"
```

Comment:
Again, I copied the behavior in /etc/conf.d/sshd.  Removed in 40b03efc13c6b5de22388db877e7cf5a6429e649